src/main/java/org/jabref/cli/Launcher.java

```diff
@@ -169,6 +170,8 @@ private static void initGlobals(PreferencesService preferences) {
         // Read list(s) of journal names and abbreviations
         Globals.journalAbbreviationRepository = JournalAbbreviationLoader
                 .loadRepository(preferences.getJournalAbbreviationPreferences());
+        Globals.predatoryJournalRepository = PredatoryJournalLoader
+                .loadRepository();
```

Comment:
This makes an online call. Please use the repository here!

Reply:
Only update will make an online call

Reply:
See https://github.com/stop-predatory-journals/stop-predatory-journals.github.io/tree/master/_data that the CSV files are not updated for **years**. Thus, it is absolutely not necessary to update the intrnal list **on each start**